napari/components/_layers/view/main.py

```diff
@@ -128,17 +125,15 @@ def dropEvent(self, event):
 
 
 class QtDivider(QFrame):
-    unselectedStlyesheet = "QFrame {;}"
-    selectedStlyesheet = """QFrame {border: 2px solid %s;
-        background-color: white; border-radius: 3px;}""" % palette['text']
-
     def __init__(self):
         super().__init__()
         self.setSelected(False)
         self.setFixedSize(50, 2)
 
     def setSelected(self, bool):
         if bool:
```

Comment:
I think we should use a different variable name here (to avoid collision with the built-in `bool` type).

Reply:
yeah that should be fixed, it isn't something that's really part of this PR, but we can make a short follow up one after

Reply:
happy to just go ahead and fix here, @kevinyamauchi do you have a preferred name? `is_selected` `state` or... ?

Reply:
went with `state` but happy to change again :)

Reply:
nevermind went with `selected`